packages/terra-hookshot/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "terra-hookshot",
+  "main": "lib/Hookshot.js",
+  "version": "0.0.0",
+  "description": "The Terra Hookshot component positions content according to a targeted attachment, ensuring they stay connected.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
```

Comment:
add some more to the key words, maybe `position, attached`? As awesome a name as hookshot is, it's not really descriptive

Reply:
Updated.